iothub/service/src/Common/TrackingHelper.cs

```diff
@@ -25,17 +33,32 @@ public static class TrackingHelper
         private const string PartitionPrefix = "-P:";
         private const string TimeStampPrefix = "-TimeStamp:";
 
+        /// <summary>
+        /// Generates a tracking Id. Not used.
+        /// </summary>
+        /// <returns>Tracking Id.</returns>
         public static string GenerateTrackingId()
         {
             return GenerateTrackingId(string.Empty, string.Empty);
         }
 
+        /// <summary>
+        /// Generates a tracking Id. Not used.
+        /// </summary>
+        /// <returns>Tracking Id.</returns>
         public static string GenerateTrackingId(string backendId, string partitionId)
         {
             string gatewayId = GatewayId;
             return GenerateTrackingId(gatewayId, backendId, partitionId);
         }
 
+        /// <summary>
+        /// Generates a unique tracking Id with gateway, backend, partition and timestamp information.
+        /// </summary>
+        /// <returns>Tracking Id.</returns>
```

Comment:
Proper English would be "A tracking Id." or "The tracking Id." In this case, I think 'a' is the right article.